src/lightning/fabric/utilities/seed.py

```diff
@@ -56,6 +60,8 @@ def seed_everything(seed: Optional[int] = None, workers: bool = False) -> int:
     np.random.seed(seed)
     torch.manual_seed(seed)
     torch.cuda.manual_seed_all(seed)
+    if XPUAccelerator.is_available():
+        XPUAccelerator.manual_seed_all(seed)
```

Comment:
I think it would be better to check device(cuda,xpu) first before calling corresponding manual_seed_all api.

Reply:
Not needed in this case, as declaration already checks.